mailpoet/assets/css/src/components-plugin/_welcome-wizard.scss

```diff
@@ -22,6 +22,12 @@
   }
 }
 
+#mailpoet-wizard-container {
+  .mailpoet-top-bar {
+    left: 0;
```

Comment:
Without adding this, the logo is positioned too close to the sidebar due to this other CSS rule that was added in a previous ticket: https://github.com/mailpoet/mailpoet/blob/3345278930871d3546601ec44a217fda461c9428/mailpoet/assets/css/src/components-plugin/_welcome-wizard.scss#L10